text-classifier/SigOpt Text Classifier Walkthrough.ipynb

```diff
@@ -0,0 +1,438 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# SigOpt for ML: Automatically Tuning Text Classifiers Walkthrough\n",
+    "*This notebook is a walkthrough companion for our blog post [SigOpt for ML: Automatically Tuning Text Classifiers](http://blog.sigopt.com/post/133089144983/sigopt-for-ml-automatically-tuning-text) by Research Engineer [Ian Dewancker](https://sigopt.com/about#ian)*\n",
+    "\n",
+    "Text classification problems appear quite often in modern information systems, and you might imagine building a small document/tweet/blogpost classifier for any number of purposes. In this example, we're building a pipeline to label [Amazon product reviews](http://www.cs.jhu.edu/~mdredze/datasets/sentiment/) as either favorable or not. First, we'll use scikit-learn to build a logistic regression classifier, performing k-fold cross validation to measure the accuracy. Next, we'll use SigOpt to tune the hyperparameters of both the feature extraction and the model building. \n",
+    "## Setup\n",
+    "If you’d like to try running this example, sign up for a [SigOpt](https://sigopt.com/) free trial and find your newly created credentials on your [profile page](sigopt.com/user/plan).  To quickly set up an environment to run this example, we recommend using a Mac OS X (with brew) or an Ubuntu machine having at least 4GB of RAM (t2.mediums in EC2 are cheap and work great).  Once you have a suitable machine this script should be all you need to get to get started:\n",
+    "```\n",
+    "# on Mac OS X or Ubuntu machine\n",
+    "git clone https://github.com/sigopt/sigopt-examples.git\n",
+    "cd sigopt-examples/text-classifier/\n",
+    "sudo ./setup_env.sh\n",
+    "```\n",
```

Comment:
Yeah, also can't run the repo cloning from the ipython notebook because to have the ipython notebook on your machine you already have the repo